### ISSUE TITLE
scan option :build_for_testing is silently ignored

### ISSUE THREAD

**Issue by lilyball** [lyndsey-ferguson/fastlane-plugin-test_center#289]
### New Issue Checklist

- [ ] Updated `fastlane-plugin-test_center` to the latest version (I checked latest source though)
- [X] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [X] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [X] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
When using `multi_scan`, the scan option `:build_for_testing` is silently ignored and it runs tests anyway. This can be seen [here](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/f7f6454ab70909a80cbfc7378b33989ac3aae542/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L235) where it simply deletes that option, after having previously inserted it [here](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/f7f6454ab70909a80cbfc7378b33989ac3aae542/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L259). It never checks to see if I originally passed that flag in.

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.156.0 | 🚫 Update available |

### Loaded fastlane plugins:

| Plugin                                    | Version | Update-Status      |
| ----------------------------------------- | ------- | ------------------ |
| fastlane-plugin-firebase_app_distribution | 0.1.4   | 🚫 Update available |
| fastlane-plugin-test_center               | 3.13.2  | 🚫 Update available |
| fastlane-plugin-sentry                    | 1.6.0   | ✅ Up-To-Date       |

**Comment by lyndsey-ferguson** [comment]
@lilyball this is fairly straightforward to fix. Just to make sure, can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again (with the `--verbose` flag)?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-289-ignored-build_without_testing"
```

If there are still problems, please let me know and attach the console output as a text file to this issue (makes it easier for me to review).  If it works, please let me know.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.14.5](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.14.5)
